### DISCUSSION TITLE
Account suspended and no response from GitHub Support for month

### DISCUSSION THREAD

**Discussion by Jewish-Art** [community/community#138206] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

Hello,

On **August 12th**, my GitHub account was suspended **without** any prior notice or warning emails. I was working on a program that interacts with the GitHub REST API, but it ended up sending too many requests, which I believe resulted in a violation of GitHub's Terms of Service, leading to an automatic suspension of my account.

That same day, I submitted a support ticket to the GitHub Support Team, but I have yet to receive any updates about my account's status.

I would like an update about the state of my account.
If you require any other information to help resolve my issue, I'm open to providing it.

I'm unsure where to go from this...

**Suspended Account:** @bitArtisan1
**Ticket ID:** 2935691

**Comment by mrrizwan304** [comment, marked as answer]
Check GitHub’s Support Documentation: Ensure you’ve followed all their guidelines for appeals.
Submit a New Support Request: Sometimes, reopening a request or submitting a new one can bring it to attention.
Reach Out on Social Media: Contact GitHub’s official Twitter support handle as a last resort.
It’s important to remain patient and persistent.

**Comment by ebndev** [comment]
Hi @Jewish-Art,

This community is managed by GitHub employees, but it is not a support forum. You have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
